rpcs3/Emu/Cell/PPUModule.cpp

```diff
@@ -853,14 +952,18 @@ std::shared_ptr<lv2_prx_t> ppu_prx_loader::load() const
 		};
 
 		// Access library information (TODO)
-		const auto& lib_info = vm::_ref<const ppu_prx_library_info>(vm::cast(segments[0] + progs[0].p_paddr - progs[0].p_offset, HERE));
-		const auto& lib_name = std::string(lib_info.name);
+		const auto& lib_info = vm::cptr<ppu_prx_library_info>(vm::cast(segments[0].first + progs[0].p_paddr - progs[0].p_offset, HERE));
+		const auto& lib_name = std::string(lib_info->name);
 
-		LOG_WARNING(LOADER, "Library %s (toc=0x%x, rtoc=0x%x):", lib_name, lib_info.toc, lib_info.toc + segments[0]);
+		LOG_WARNING(LOADER, "Library %s (rtoc=0x%x):", lib_name, lib_info->toc);
 
-		prx->specials = ppu_load_exports(link, lib_info.exports_start, lib_info.exports_end);
+		prx->specials = ppu_load_exports(link, lib_info->exports_start, lib_info->exports_end);
 
-		ppu_load_imports(link, lib_info.imports_start, lib_info.imports_end);
+		const u32 min_addr0 = ppu_load_imports(link, lib_info->imports_start, lib_info->imports_end);
+		const u32 min_addr = std::min<u32>({ lib_info.addr(), +lib_info->imports_start, +lib_info->exports_start, min_addr0 });
```

Comment:
It does not change the value, but type instead. It's slightly useful for be_t to cast it to native integral type, but doesn't work well there for std::min for some reason.